pyproject.toml

```diff
@@ -40,8 +38,8 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.7"
-fastapi = ">=0.61"
+python = "^3.9"
```

Comment:
We only test with python 3.9+, so I removed support for lower version. (If we don't test with it, we don't support it)